.wordlist.txt

```diff
@@ -45,3 +45,7 @@ UUID
 VM
 YAML
 enablement
+Quickstart
+http
+rst
+toctree
```

Comment:
These aren't words and shouldn't be in here.
Remember that this is the word list that every user of the starter pack will be starting with.